spikingjelly/activation_based/triton_kernel/flexsn/wrapper.py

```
@@ -15,7 +15,14 @@
 from .info import FlexSNInfo
 
 
-__all__ = ["flexsn_inference", "flexsn_inference_final_state", "flexsn_forward", "flexsn_backward", "flexsn_backward_ncl_bucket", "FlexSNFunction"]
+__all__ = [
+    "flexsn_inference",
+    "flexsn_inference_final_state",
+    "flexsn_forward",
+    "flexsn_backward",
+    "flexsn_backward_ncl_bucket",
+    "FlexSNFunction",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)